### PR TITLE
fix: reduce kde image size

### PR DESCRIPTION
**Pull request by madonuko**

### CODE REVIEW

**Comment by lleyton** [comment]
wonder what is pulling this in the first place?

**Comment by madonuko** [comment]
> wonder what is pulling this in the first place?

on my kde system:
```
$ dnf rq --whatrequires proj --installed --recursive
digikam-0:8.4.0-1.fc40.x86_64
digikam-libs-0:8.4.0-1.fc40.x86_64
gdal-libs-0:3.8.5-2.fc40.x86_64
libgeotiff-0:1.7.1-12.fc40.x86_64
libspatialite-0:5.1.0-5.fc40.x86_64
opencv-imgcodecs-0:4.9.0-3.fc40.x86_64
```

File: katsu/modules/kde/kde.yaml [hunk 1]
```diff
@@ -4,3 +4,13 @@ dnf:
   packages:
     #beta - still need to set up our own comps
     - "@ultramarine-kde-product-environment"
+  exclude:
+    - 'proj-data-*'
+    - ibus
```
Comment (lleyton) [inline katsu/modules/kde/kde.yaml]: can this be fixed in the comp group?

Comment (madonuko) [reply katsu/modules/kde/kde.yaml]: How?

Comment (lleyton) [reply katsu/modules/kde/kde.yaml]: I thought ibus was being pulled in due to a sub package in a common comp?

Comment (madonuko) [reply katsu/modules/kde/kde.yaml]: Well it gets pulled in everywhere but all of them are weak deps (except plasma-desktop and sddm which requires ibus-libs for whatever reasons)

**Comment by lleyton** [comment]
as for proj data, Jade slimmed down the KDE comps a lot, so we'll see if that makes it such that it doesn't pull that shit in

**Comment by lleyton** [comment]
proj data isn't pulled in anymore, so no need for exclude

as for ibus, it's small enough to keep it IMO

**Comment by madonuko** [comment]
> as for ibus, it's small enough to keep it IMO

image

**Comment by korewaChino** [comment]
why is it pulling in virtualbox

**Review by lleyton — APPROVED** [review]

**Pull request merged**